src/plugins/irc-events/message.js

```diff
@@ -95,5 +96,27 @@ module.exports = function(irc, network) {
 		if ([Msg.Type.MESSAGE, Msg.Type.ACTION].indexOf(data.type) !== -1) {
 			LinkPrefetch(client, chan, msg);
 		}
+
+		// TODO: For now, do not send notifications if any client is attached
+		if (highlight
+		&& (!data.time || data.time > Date.now() - 2.16e+7) // Message not older than 6 hours
```

Comment:
Why? Arbitrary, or is there a reason for this? What if we have an issue with timezones?